### PR TITLE
feat(api): Add the JavaScript File Export Options

### PR DESCRIPTION
**Pull request by akashpambhar**

### CODE REVIEW

File: src/main/java/com/crowdin/client/sourcefiles/model/JavaScriptFileExportOptions.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.crowdin.client.sourcefiles.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = false)
+public class JavaScriptFileExportOptions extends ExportOptions {
+
+    private String exportQuotes;
```
Comment (andrii-bodnar) [inline src/main/java/com/crowdin/client/sourcefiles/model/JavaScriptFileExportOptions.java:10]: please create an enum for this type

**Review by andrii-bodnar — COMMENTED** [review]
@akashpambhar thanks for the contribution!

**Review by andrii-bodnar — APPROVED** [review]
@akashpambhar thank you!

**Comment by akashpambhar** [comment]
@andrii-bodnar thank you for the opportunity! This was my first contribution in Open Source projects.

**Pull request merged**